### PR TITLE
fix ctrl+click on windows

### PR DESCRIPTION
**Pull request by bridiver**
also fixes a few other minor issues like links not re-using named frames

### CODE REVIEW

**Comment by bridiver** [comment]
fixes browser-laptop #412

File: atom/renderer/api/atom_api_web_frame.cc [hunk 1]
```diff
@@ -75,7 +77,15 @@ void WebFrame::RegisterElementResizeCallback(
 }
 
 void WebFrame::AttachGuest(int id) {
+  // This is a workaround for a strange issue on windows with background tabs
+  // libchromiumcontent doesn't appear to be making the check for
+  // params.dispostion == NEW_BACKGROUND_TAB in WebContentsImpl
+  // This results in the BrowserPluginGuest trying to access the native
+  // window before it's actually ready. This hack works around that
+  // by always delaying the access
+  content::BrowserPluginManager::Get()->GetBrowserPlugin(id)->updateVisibility(false);
```
Comment (bbondy) [inline atom/renderer/api/atom_api_web_frame.cc]: Wow this was a strange one, nice job.  I don't think I've ever run into something like this but I've always compiled all of my own source on my projects and not used prebuilt stuff.

**Comment by bbondy** [comment]
++ merging

**Comment by BrendanEich** [comment]
Nit: "dispostion" typo in comment.

Great to have this. Is there another issue on file to clean up at a higher level, so the code doesn't have extra tests that seem to mix concerns and layers? Thanks.

**Comment by bridiver** [comment]
It's actually not too terrible. "AttachGuest" is attaching the browser plugin so flipping the visibility there isn't great, but it's not the worst thing in the world. I haven't tried it yet with a fresh build of libchromiumcontent, but another possible reason this is happening is because all the tabs have the same "embedder" and it is visible even when the guest will not be. I'm definitely going to look into more to verify the underlying issue now that the immediate problem is solved.

**Pull request merged**